app/build.gradle

```diff
@@ -439,7 +439,7 @@ task printGeckoviewVersions {
     }
 }
 
-apply from: 'https://github.com/mozilla-mobile/android-components/raw/v' + Versions.mozilla_android_components + '/components/service/glean/scripts/sdk_generator.gradle'
+apply from: 'https://github.com/mozilla-mobile/android-components/raw/master/components/service/glean/scripts/sdk_generator.gradle'
```

Comment:
I didn't follow this discussion. Did the glean folks recommend going back to master?

Reply:
Since this is going back to track a SNAPSHOT, and there's no way to get the git hash or tag from a snapshot, going back to master is probably the closest thing to workable we have (short of fixing this in a more fundamental way).